docs/features/distribution-metrics.md

```diff
@@ -31,8 +32,4 @@ Step 2: Set the Time Filter field name to `@timestamp` with the pulldown option.
 
 ![Image of Step 2 configuring kibana console](assets/kibana-step2.png)
```

Comment:
Not for this PR, but I'm going to circle back on all of our alt text. This really isn't descriptive enough

Reply:
Agreed, fixing that (and then falling down that rabbit hole) felt like scope creep on the ticket

Reply:
I added it to our doc

Reply:
👍